src/classes/ALLO_MakeDefaultAllocations_BATCH.cls

```diff
@@ -39,43 +39,40 @@ public class ALLO_MakeDefaultAllocations_BATCH implements Database.Batchable<sOb
 
     /** @description Allocations settings object.*/
     private Allocations_Settings__c alloSettings = UTIL_CustomSettingsFacade.getAllocationsSettings();
+    
+    /** @description Which opportunity types to exclude. We need to split this into a set 
+    * to avoid a partial match, e.g. Donor could be matched to the string Matching Donor*/
+    private Set<String> oppTypesToExclude = String.isBlank(alloSettings.Excluded_Opp_Types__c) ? new Set<String>() : new Set<String>(alloSettings.Excluded_Opp_Types__c.split(';'));
+
     /** @description The query for the batch process to run on.*/
     private String query;
     
     /** @description The batch process constructor; creates the query based on type and record type exclusion settings.*/
```

Comment:
@njjc should we update the constructor comment explaining why we are using this approach and no longer filtering via the query?